charts/gitprime-app/v4.0.0/questions.yml

```diff
@@ -494,6 +494,71 @@ questions:
     default: "INFO"
 
 
+
+  # Batch Worker Settings
+  - variable: dataPipeline.batchWorker.replicaCount
+    type: string
+    required: true
+    group: "Data Pipeline / Batch Workers"
+    label: Number of Pods for Batch Workers
+    description: The number of pods to deploy for the batch workers.
+    default: 1
```

Comment:
I *think* we only ever want 1 of the batch workers.  @ntaylor-gitprime should we hard code the helm chart to only ever allow a single pod of the batch worker?

Reply:
Yes, I can't think of a scenario where we would want multiple JVMs running the same batch processes.